core/sts-3.3.md

```diff
@@ -108,7 +108,19 @@ unable to do anything about except retrying at a later time.
 This is to ensure that it is not possible to fool users into allowing a
 man-in-the-middle attack.
 
-## Security
+## Security considerations
+
+### Acceptance of STS policy
+
+STS policies SHOULD ONLY be accepted for connections where the TLS certificate
+chain can be validated entirely, to either a user-specified root or the system
+root certificate bundle.
+
+### Visual feedback of STS upgraded connections
+
+Clients SHOULD NOT display visual feedback indicating that an STS-upgraded TLS
+connection has any enhanced security over a plaintext connection unless a user
+explicitly trusts the certificate.
```

Comment:
The precise details of how the client should represent untrusted vs implicit trust vs explicit trust is up to the clients, but I would look at best practices with browser UI regarding security for inspiration.